book.rb

```diff
@@ -0,0 +1,17 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(label, author)
+    @title = label
+    @author = author
+    @rentals = []
+  end
+
+  def add_rentals(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] Please edit this method to make it create a new rental and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.
[Hint]: You can do it like that:
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```